package.json

```diff
@@ -32,10 +32,15 @@
   },
   "homepage": "https://github.com/ConsenSys/Tokens#readme",
   "dependencies": {
-    "truffle": "4.0.1",
+    "truffle": "4.0.4",
     "truffle-hdwallet-provider": "0.0.3"
   },
   "devDependencies": {
+    "babel-polyfill": "^6.26.0",
```

Comment:
Please pin the versions.

Reply:
Why introduce babel at all? It doesn't seem to actually be used anywhere.

Reply:
It's for the assertRevert.js that uses ES6. It needs to be transpiled.

Reply:
Unless I have it wrong? Maybe one can remove it somehow? But it was needed for adding assertRevert. Do you know how to change both to not have to use Babel? I admit, transpilers always confuzzle me.

Reply:
What needs to be transpiled? I don't see any super exotic JS in assertRevert.js

Reply:
What Node JS version are you using? We develop against Node 8 for this repo.

Reply:
Maybe it's just the import statement then in the tests? vs require? Lemme try that.